README.md

```diff
@@ -13,3 +13,31 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+Bullet 1
+
+1. Asymptotic analysis is theoretical, as it describes algorithm behavior as the input reaches infinity.  Actual performance is empirical, as it describes algorithm behavior with a specific set of inputs.  The two are different, as actual performance depends on constants and machine performance, whereas asymptotic analysis ignores those factors.
```

Comment:
The input doesn't need to reach infinity -- it never will. What matters is the value of $n_0$.